example-application/api.js

```diff
@@ -106,6 +99,20 @@ const defineRoutes = (expressApp) => {
     res.status(204).end();
   });
 
+  expressApp.use((req, res, next) => {
+    const start = Date.now();
+
+    res.on('finish', async () => {
```

Comment:
Minor - The global (all paths) middlewares are defined now in initializeWebServer, this function 'define routes' is for specific route. I would consider moving this function to where bodyparser is defined